### DISCUSSION TITLE
Extend tapOn or repeat functionality to enable doubleTaps or more

### DISCUSSION THREAD

**Discussion by VNDRN** [mobile-dev-inc/Maestro#674] [Ideas]
In our app, we have a setting that's hidden by default. You can enable this setting by tapping n amount of times on an element within a certain period (eg. 2000ms). Whilst creating a flow for this, I noticed that with regular `tapOn`, or with `repeat`, the taps are not executed fast enough. 

It would be great if there was an option to execute double taps, triple taps, or in general nTaps. Is this something that could easily be added? 

Or is there already a workaround that would solve my issue?

**Comment by ArthurSav** [comment]
Hi @VNDRN this is a great suggestion. Also mentioned [here](https://github.com/mobile-dev-inc/maestro/issues/1090). We'll try to implement when possible.
